runtime/acala/src/lib.rs

```diff
@@ -129,7 +129,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	apis: RUNTIME_API_VERSIONS,
 	#[cfg(feature = "disable-runtime-api")]
 	apis: sp_version::create_apis_vec![[]],
-	transaction_version: 2,
+	transaction_version: 3,
```

Comment:
no need to bump this. no user should call system pallet (except remark). I don't want to break ledger just to ensure remark is working.

Reply:
https://github.com/paritytech/substrate/pull/13207

Reply:
revert